### DISCUSSION TITLE
Checks status UI broken

### DISCUSSION THREAD

**Discussion by ahouseago** [community/community#15323] [Other Feature Feedback, Questions, & Ideas]
I am on Firefox 99.0.1 and the UI for seeing the status of code checks on commits is broken. Instead of appearing as an overlay it's nested in the commit row so the height of the box is too small. Here is an example:

![Screenshot 2022-04-21 at 12 32 51](https://user-images.githubusercontent.com/41902631/164450145-c5b30e21-546a-4081-b9de-ef8b4902321d.png)

This used to work fine so I expect is the result of a recent change.

EDIT: I have managed to work around it by disabling the `overflow: auto !important` style applied by the `.overflow-auto` class on the first div child of the li representing the commit in dev tools.

**Comment by queenofcorgis** [comment]
@ahouseago thanks for sharing you workaround, I've added the corresponding labels so the proper team can easily find this 👍🏼
